x/ethbridge/client/rest/rest.go

```diff
@@ -55,6 +55,15 @@ type burnOrLockEthReq struct {
 	CrosschainFee     sdk.Int      `json:"cross_chain_fee_amount" yaml:"cross_chain_fee_amount"`
 }
 
+type signProphecyReq struct {
+	BaseReq               rest.BaseReq `json:"base_req"`
+	NetworkDescriptor     string       `json:"network_descriptor"`
+	CosmosSender          string       `json:"cosmos_sender"`
+	EthereumSignerAddress string       `json:"ethereum_signer_address"`
+	Signature             string       `json:"signature"`
+	ProphecyID            string       `json:"prophecy_id"`
```

Comment:
With the new data structure on the cosmos side, we are going to have to add more to this struct. We can do this after this PR is merged.

Reply:
Ok, will sync up the change after smart contract finalied.